tests/Tutor.Infrastructure.Tests/TestData/EventStore/EventQueryable/EventQueryableTestCaseGenerator.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using static Tutor.Infrastructure.Tests.TestData.EventStore.EventQueryableTestCase;
+
+namespace Tutor.Infrastructure.Tests.TestData.EventStore.EventQueryable
+{
+    public static class EventQueryableTestCaseGenerator
+    {
+        private static IEnumerable<object[]> _testCases = new List<object[]>();
+
+        public static IEnumerable<object[]> GenerateTestCases(IEnumerable<QueryParameter> parameters, int amountPerParameter)
+        {
+            if (!_testCases.Any())
```

Comment:
Can be inverted to reduce nesting.

Reply:
Corrected in 02acfa3104adbf25dc1aaf41773e57833916d87c.